packages/jest-haste-map/src/ModuleMap.ts

```diff
@@ -32,10 +32,15 @@ export type SerializableModuleMap = {
 };
 
 export default class ModuleMap {
+  public readonly uniqueID: number;
   private readonly _raw: RawModuleMap;
+  private json: SerializableModuleMap | undefined;
   static DuplicateHasteCandidatesError: typeof DuplicateHasteCandidatesError;
+  private static nextUniqueID = 0;
```

Comment:
`static` above instance variables?

Also, as mentioned somewhere else, start this at `1` so it's not falsy?

Reply:
static above instance should be a lint rule / auto-fixable? done.

Reply:
>static above instance should be a lint rule / auto-fixable?

yeah, probably 🙂